scripts/20160812-col429/add_activity_enums.sql

```diff
@@ -0,0 +1,16 @@
+-- Add new activity enums
+
+ALTER TYPE enum_activities_type ADD VALUE 'view_asset';
+ALTER TYPE enum_activities_type ADD VALUE 'whiteboard_add_asset';
+ALTER TYPE enum_activities_type ADD VALUE 'whiteboard_chat';
+
+-- Add new activity type enums
+
+ALTER TYPE enum_activity_types_type ADD VALUE 'view_asset';
```

Comment:
@paulkerschen : Do you know what the difference is between `enum_activities_type` and `enum_activity_types_type` ?